### PR TITLE
Utilize TitanUtil's PathFinder

### PR DESCRIPTION
**Pull request by RyanHir**
Note: TitanUtil's PathFinder is a wrapper to WPI PathWeaver

I am adding @democat3457 as a reviewer as we will need more than one person to understand the changes in robot code, you don't have to fully understand the internals of the TitanUtil functions.

This code should not be merged until the team has a working demo on the physical robot.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -126,34 +129,50 @@
...
     public static final double HOPPER_LEFT_SPEED = 0.85;
     public static final double HOPPER_RIGHT_SPEED = 0.7;
-    
+
+    // ================================================================================
+    // PathWeaver Data
+    // ================================================================================
+
+    public static final String DRIVEBASE_PATHWEAVER_PATH = "";
+    public static final DriveConfig DRIVEBASE_PATHWEAVER_CONFIG = new DriveConfig.Builder() //
```
Comment (democat3457) [inline src/main/java/frc/robot/Constants.java:157]: Are we not going to use the other constants that were in the Auto/Drive Constant files?

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -22,7 +21,6 @@
 
   @Override
   public void robotInit() {
-    Logger.DEBUG = true;
```
Comment (democat3457) [inline src/main/java/frc/robot/Robot.java:25]: Why is `DEBUG` off?

File: src/main/java/frc/robot/subsystems/Drivebase.java [hunk 3]
```diff
@@ -181,11 +181,23 @@ public void setSlot(int slot) {
         right.selectProfileSlot(slot, 0);
     }
 
+    /**
+     * @deprecated use driveTank
```
Comment (democat3457) [inline src/main/java/frc/robot/subsystems/Drivebase.java:185]: Why was this deprecated?

File: src/main/java/frc/robot/subsystems/Drivebase.java [hunk 4]
```diff
@@ -222,23 +241,10 @@ public void setRamping(double ramping) {
         right.configClosedloopRamp(0);
     }
 
-    public double getRamping() {
```
Comment (democat3457) [inline src/main/java/frc/robot/subsystems/Drivebase.java:225]: Is this method just unused (and thus unnecessary)?

Comment (RyanHir) [reply src/main/java/frc/robot/subsystems/Drivebase.java:225]: It is unused IIRC and was there for our old MotionMagic code

File: src/main/java/frc/robot/subsystems/Drivebase.java (hunk 3 above)
Comment (RyanHir) [reply src/main/java/frc/robot/subsystems/Drivebase.java:185]: There are now prewritten functions in the abstract class which do the same

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (RyanHir) [reply src/main/java/frc/robot/Robot.java:25]: I deprecated `Logger.DEBUG` and is unused. I deprecated it as the logs can just be ignored during Comp so no reason to have a flag to disable it

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (RyanHir) [reply src/main/java/frc/robot/Constants.java:157]: This will be discussed at the meeting

**Pull request merged**